usr/sbin/regionsrv-enabler-azure

```diff
@@ -12,6 +12,35 @@
 # Lesser General Public License for more details.
 # You should have received a copy of the GNU Lesser General Public
 # License along with this library.
+
+"""
+Check the different states of the license type:
+    START   LICENSE CHANGED TO     CACHE   ACTION
+    BYOS-1  SLES                   No      Register - Update Infra
+    BYOS-1  SLES                   BYOS    Register - Update Infra
```

Comment:
I think `SLES` should be `PAYG` the state transitions are product independent and apply equally to SLES For SAP and SLE HPC. Mixing a license state with product nomenclature is confusing.